src/Common/Hooks/useUrlFilters/types.ts

```diff
@@ -32,6 +32,10 @@ export type UseUrlFiltersReturnType<T, K = unknown> = K & {
      * Currently applied page size
      */
     pageSize: number
+    /**
+     * PageNumber for the current page
+     */
+    pageNumber?: number
```

Comment:
We don't need the page number to be returned. It is just for internal state calculation and abstracted from the consumer component